spec/unit/providers/aws/credentials_spec.rb

```diff
@@ -1,45 +1,47 @@
 require 'chef/provisioning/fog_driver/providers/aws/credentials'
 
 describe Chef::Provisioning::FogDriver::Providers::AWS::Credentials do
-  describe "#load_ini" do
+  subject { Chef::Provisioning::FogDriver::Providers::AWS::Credentials.new }
```

Comment:
I actually prefer something like `let(:credentials) { Chef::Provisioning::FogDriver::Providers::AWS::Credentials.new }` as opposed to the `subject` magic.  I think it is more clear.  Can you make that change?